### PR TITLE
Use -wordlist instead of -file as an option to define domain(s) dictionary

### PR DESCRIPTION
**Pull request by bl4de**
Hi @wdahlenburg ,

I hope you are well! 
First of all - thank you for a great tutorial (https://wya.pl/2022/06/16/virtual-hosting-a-well-forgotten-enumeration-technique/) as well as for publishing your very handy tool 😉 

This PR is a proposition to change one of the options name to something more common and "familiar" across other tools. Feel free to throw it out of the window if you don't like it 😄 

I have tested the tool with my change and it works well, I have updated README as well.

Stay Safe!

bl4de

### CODE REVIEW

File: utils/file_utils.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import (
 	"os"
 )
 
-func ReadDomains(filename string) ([]string, error) {
+func ReadDomains(Wordlist string) ([]string, error) {
```
Comment (wdahlenburg) [inline utils/file_utils.go]: Can this be changed to lowercase `wordlist` for consistency?

**Review by wdahlenburg — CHANGES_REQUESTED** [review]
Hi @bl4de. Thanks for submitting this. I agree that wordlist is a better argument name than file.

Can you make the one small change and then I'm good to have this merged? I'll bump the version once it's done.

File: utils/file_utils.go (hunk 1 above)
Comment (bl4de) [reply utils/file_utils.go]: Hi @wdahlenburg,

Good point, done!

**Review by wdahlenburg — APPROVED** [review]
Looks great!

**Pull request merged**